portfolio/src/main/webapp/script.js

```diff
@@ -15,3 +15,51 @@
 /**
  * Adds a random greeting to the page.
  */
+
+
+
+async function getContentUsingAsyncAwait() {
+  console.log('It\'s starting to fetch!');
+
+  const response = await fetch('/data');
+
+  const content = await response.text();
+
+  document.getElementById('info_container').innerText = content;
+}
+
+function getServerComments() {
+  fetch('/data').then(response => response.json()).then((data) => {
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
+    // const divElement = document.getElementById('info_container');
+
+    if (data.length === 0) {
+      return;
+    }
+    console.log(data);
+
+
+
+
+    // Build the list of history entries.
+    const historyEl = document.getElementById('info_container');
+    var counter = data.length;
+    for (let start = 0; start < data.length; start++) {
```

Comment:
This loop can be simplified by using a for...of loop (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)

```
for (const comment of data) {
  ...
}
```